trap/src/main/java/com/cursorinsight/trap/datasource/hardware/TrapBatteryCollector.kt

```diff
@@ -0,0 +1,134 @@
+package com.cursorinsight.trap.datasource.hardware
+
+import android.app.Activity
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Context.BATTERY_SERVICE
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.BatteryManager
+import android.os.Build
+import android.util.Log
+import com.cursorinsight.trap.TrapConfig
+import com.cursorinsight.trap.datasource.TrapDatasource
+import com.cursorinsight.trap.datasource.sensor.TrapAccelerometerCollector
+import com.cursorinsight.trap.util.TrapTime
+import org.apache.commons.collections4.queue.SynchronizedQueue
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Monitors for battery status then reports them
+ * as data frames.
+ *
+ * @property storage The data frame queue.
+ * @constructor
+ * Sets up the data collector.
+ *
+ * @param config The library config instance.
+ */
+class TrapBatteryCollector(
+    private val storage: SynchronizedQueue<JSONArray>,
+    @Suppress("UNUSED_PARAMETER") config: TrapConfig,
+): TrapDatasource {
+    /**
+     * The Trap event type for wifi
```

Comment:
C&P error, and later too.

Reply:
OK, I found the fix for these in one of the following commits -- though IMO one "wifi" remained. :)